migratekvdb/helper.go

```diff
@@ -0,0 +1,28 @@
+package migratekvdb
+
+import (
+	"encoding/hex"
+)
+
+// loggableKeyName returns a string representation of a bucket key suitable for
+// logging.
+func loggableKeyName(key []byte) string {
+	// For known bucket names, return as string if printable ASCII.
+	if isPrintableASCII(key) {
+		return string(key)
+	}
+
+	// Otherwise return hex encoding.
+	return hex.EncodeToString(key)
```

Comment:
nit: i wonder if we should prefix the hex string with say `x` so we know it's a binary blob encoded as hex and not some weird bucket name.

Reply:
Yeah works, but really this is a nit, not super important imo.